server/src/main/java/com/defold/extender/services/CocoaPodsService.java

```diff
@@ -236,7 +248,10 @@ public class PodSpec {
         public PlatformSet resources = new PlatformSet();
         public PlatformSet frameworks = new PlatformSet();
         public PlatformSet libraries = new PlatformSet();
+        public PlatformSet publicHeaders = new PlatformSet();
         public File dir;
+        public boolean installed;
```

Comment:
This field is true if the pod has been installed. Sometimes a Podspec contains subspecs which may not be installed, depending on Podspec settings. Handled while Cocoapods is resolving dependencies.